app/views/schools/usage/_within_school_card.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="card <%= current_supply %>-card" id="within-school" style="display: none;">
+  <div class="card-header"></div>
+  <div class="card-body">
+    <h4 class="card-title">
+      <%= date_option %>
+    </h4>
+    <div class="form-group">
+      <div class="input-group date <%= 'week-view' if week_picker %>" id="datetimepicker3" data-target-input="nearest">
```

Comment:
Do we need to render this in full rather than using the customised date picker simple form thingy?

Reply:
I would have to convert the whole (or part of?) the form to simple form to get access to those, I think. Can look at it if you think it's worth it.